### ISSUE TITLE
[kubernetes] Helm chart issues, super easy to reproduce

### ISSUE THREAD

**Issue by DarkBlaez** [gravitee-io/issues#4262]
The current helm chart for apim3-3.0.10.tgz fails immediately. This error would have been immediate. To clarify, this takes place in the updated helm repo, this also happens with the version pulled from git.

Error: template: apim3/templates/api/api-deployment.yaml:29:28: executing "apim3/templates/api/api-deployment.yaml" at <include (print $.Template.BasePath "/api/api-configmap.yaml") .>: error calling include: template: apim3/templates/api/api-configmap.yaml:229:36: executing "apim3/templates/api/api-configmap.yaml" at <.Values.notifiers.smtp.starttls.enabled>: nil pointer evaluating interface {}.enabled

Please advise? It appears the file change check is blowing errors. While this should work, not sure why it would error out upon a fresh install. As in, it's not even executing the chart, just straight to an error.

Also, a bit of advice, unification on terms would be nice.... i.e starttlsEnable vs starttls.enabled as your attributes. Which should you settle on, almost as if two different individuals worked on the same section of the values/configmap. Settle on one or the other

```yaml
  enabled: true
  host: smtp.example.com
  port: 25
  from: info@example.com
  username: info@example.com
  password: example.com
  subject: "[gravitee] %s"
  properties:
    auth: true
    starttlsEnable: false
    #localhost: apim.example.com

notifiers:
  smtp:
    enabled: true
    host: ${email.host}
    subject: ${email.subject}
    port: ${email.port}
    from: ${email.from}
    username: ${email.username}
    password: ${email.password}
    starttls.enabled: false
    #    ssl:
    #      trustAll: false
    #      keyStore:
    #      keyStorePassword:
```

Thanks
DB

**Comment by DarkBlaez** [comment]
Try to make the chart more consistent. Look at the smtp sections. In the configmap you call our email: but in the values.yaml you reference smtp: as the root. Consistency goes a very long way to maintainability. Standard naming conventions. Email and Notifications are the context, smtp is the mechanism. 

smtp:
  enabled: false
  host: smtp.example.com
  port: 25
  from: info@example.com
  username: info@example.com
  password: example.com
  subject: "[gravitee] %s"
  properties:
    auth: true
    starttlsEnable: false
    #localhost: apim.example.com

notifiers:
  smtp:
    enabled: true
    host: ${email.host}
    subject: ${email.subject}
    port: ${email.port}
    from: ${email.from}
    username: ${email.username}
    password: ${email.password}
    starttls.enabled: false
    ssl:
      trustAll: false
      keyStore:
      keyStorePassword:

**Comment by brasseld** [comment]
Hi DB,

Thanks for your feedback.
I'm a bit surprised by that one but I'll have a look asap.

Maybe you can propose a PR with the expected changes ?

Regards,

**Comment by DarkBlaez** [comment]
I prefer to adjust my variant locally to test and indicate the change as merely an observation and a fix. I'll monitor the changes to see if the recommendation pans out. I was able to get things installed with the changes I made. The starttls.enabled should really be changed to sync with the prior naming and to function. The other recommendation is merely something I instruct my teams on to consider what the context is separate from the mechanism in their naming. 

Kind regards
DB
